SPECS/iscsi-initiator-utils/iscsi-initiator-utils.spec

```diff
@@ -1,244 +1,215 @@
-%global __provides_exclude_from ^(%{python2_sitearch}/.*\\.so|%{python3_sitearch}/.*\\.so)$
+%global commit0			c26218d9f8afdca44a492a4c3811648bd2880b26
 
-# Use only 7-10 first characters of the git commit hash!
-%global git_commit            2a8f9d81d0d6b5094c3fe9c686e2afb2ec27058a
-%global git_short_commit      2a8f9d8
-%global git_short_commit_date 20210729
-%global open_iscsi_build      4
-%global open_iscsi_version    2.1
+# Disable python2 build by default
+%bcond python2 0
+%bcond python3 1
 
 Summary:        iSCSI daemon and utility programs
 Name:           iscsi-initiator-utils
-Version:        6.%{open_iscsi_version}.%{open_iscsi_build}+%{git_short_commit_date}.%{git_short_commit}
-Release:        3%{?dist}
+Version:        2.1.9
```

Comment:
The previous version seems to be `6.x` and we are moving to `2.x`.
I am not an expert here, so can you please give a little context.

Reply:
the comments in the spreadsheet said to normalize the version.  the version of open-iscsi that was previously used was 2.1.4 (not 6.*), i think 6.* versioning came from the fedora spec that was used as a basis.  i tried to normalize to open-iscsi version (2.1.9).

Reply:
@PawelWMS , can you provide clarity around what the versioning for this should be moving forward?

Reply:
I am supportive of this change.  Since 2.0->3.0 is a breaking change, this gives us the chance to resolve.
If we wanted to be careful here, we could add the "Epoch" value to the version so that 1:2.1.9 supercedes 6.gack, but we don't offer upgrades from Mariner 2.0 to 3.0, so we're ok without the Epoch